### ISSUE TITLE
Set rclone logging level via RPC

### ISSUE THREAD

**Issue by arttkach** [rclone/rclone#7439]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/set-rclone-logging-level-via-rpc/42687/12

#### What is the problem you are having with rclone?

Unable to set loggingLevel using librclone.RPC

#### What is your rclone version (output from `rclone version`)

librclone.RPC is used

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Win11 64 bit

#### Which cloud storage system are you using? (e.g. Google Drive)

AWS S3

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

Read the forum post. Link to the post is provided at the very top.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by nielash** [comment]
Is it maybe just this? https://github.com/nielash/rclone/commit/e42aeb9670ff7e9e90b53825e77ff4d42c586a53

Seems to be [working for me now](https://github.com/nielash/rclone/commit/c1a1ce76f3b9f693e735d017909b9dcff353761a#diff-e0835d567a237a2c9f0ebd7fcbc8ff62b81fe728c7d973fc03df20a176d5e853R24-R27) after adding that one line.

**Comment by arttkach** [comment]
@nielash Have you tried setting logging via librclone.RPC like I tried in my [post](https://forum.rclone.org/t/set-rclone-logging-level-via-rpc/42687/12)?

**Comment by nielash** [comment]
@arttkach, yes -- I reproduced your bug and verified that adding [this line](https://github.com/nielash/rclone/commit/e42aeb9670ff7e9e90b53825e77ff4d42c586a53) fixes it. As for whether that is the "correct" fix, I defer to @ncw 🙂 

My working example code is linked above, in which I am overriding the default `LogLevel` from `NOTICE` to `INFO` on [this line](https://github.com/nielash/rclone/blob/c1a1ce76f3b9f693e735d017909b9dcff353761a/cmd/bisync/librclonebisync/example/example.go#L25). Note that I am using `ci.LogLevel = 6` because this field in my struct is of type `fs.LogLevel`, but I have tested that it also works to define it as `string` (in which case `"INFO"` would work but `"6"` would not.)

**Comment by Its-Just-Nans** [comment]
Can't you just

```rust
rpc("options/set", json!({
    "main": {
         "LogLevel": "INFO"
    }
}))
```

?
